documentation/assemblies/overview/assembly-kafka-connect-components.adoc

```diff
@@ -0,0 +1,30 @@
+// This assembly is included in:
+//
+// overview/overview.adoc
+
+[id="kafka-connect-components_{context}"]
+= About Kafka Connect
+
+[role="_abstract"]
+Kafka Connect is an integration toolkit for streaming data between Kafka brokers and other systems.
+The other system is typically an external data source or target, such as a database.
+
+Kafka Connect uses a plugin architecture.
+Plugins allow connections to other systems and provide additional configuration to manipulate data.
+Plugins include _connectors_ and other components, such as data converters and transforms.
+A connector operates with a specific type of external system.
+Each connector defines a schema for its configuration.
+You supply the configuration to Kafka Connect to create a _connector instance_ within Kafka Connect.
+Connector instances define a set of tasks to move data between systems.
+
+Strimzi operates Kafka Connect in _distributed mode_, distributing data streaming tasks across worker pods.
+A Kafka Connect cluster comprises a group of worker pods.
+A single worker instantiates a connector, but tasks are distributed across the group.
```

Comment:
Changed to:

> A single worker instantiates a connector.
> Each connector comprises one or more tasks that are distributed across the group of workers.

Reply:
I saw this `A single worker instantiates a connector.` on few other places as well, but I'm not sure it will be clear to the users what it means. Technically, one of the workers will be a leader and will create the connector. So I guess this might be technically correct. But without explaining this, I'm not sure what would the user / reader understand under it.